aten/src/ATen/native/native_functions.yaml

```diff
@@ -6711,13 +6711,33 @@
     CPU, CUDA: min
     QuantizedCPU: min_quantized_cpu
 
+- func: fmin(Tensor self, Tensor other) -> Tensor
+  use_c10_dispatcher: full
+  variants: method, function
+  dispatch:
+    Math: fmin
+
+- func: fmin.out(Tensor self, Tensor other, *, Tensor(a!) out) -> Tensor(a!)
+  dispatch:
```

Comment:
.out variants now need to be c10 compliant by having the `use_c10_dispatcher: full` option and in the C++ implementation the out tensor must be the last parameter.

Reply:
Addressed!